esaco-app/src/test/java/it/infn/mw/esaco/test/utils/EsacoTestUtils.java

```diff
@@ -28,7 +28,8 @@
   protected final String TOKEN_FROM_UNKNOWN_ISSUER =
       "eyJraWQiOiJyc2ExIiwiYWxnIjoiUlMyNTYifQ.eyJzdWIiOiI3M2YxNmQ5My0yNDQxLTRhNTAtODhmZi04NTM2MGQ3OGM2YjUiLCJpc3MiOiJodHRwczpcL1wvaWFtLmxvY2FsLmlvXC8iLCJleHAiOjE1MTA1Nzg1NDksImlhdCI6MTUxMDU3NDk0OSwianRpIjoiMWNiMDYxMWItNGZlYS00NjRhLWJiNDktMzllNzE1MzFkMmJjIn0.cFXl8zJQUqLEF2kuaxx_w4znm4rMGlGguN_01cs5CSKV6FMPCVGLzDeDXg068uQWtExINBOzdtRlzOgFR5-s-9XFlaQBBmsjsuZBMwPlQvh-ceQGmAuEZt-QmU-kh7zarAsa2N4wkExFdP7iB6Mz8RFjBN3OPA5puMOpDumCQSQ";
 
```

Comment:
The target of this issue is to allow and propagate introspection responses which contain more info than IamIntrospection object. Is there a test with a JWT that contains some fields not included in IamIntrospection object? In order to check if it has been preserved and returned by Esaco introspect endpoint.
You can check the JSON returned using something like:
`.andExpect(jsonPath("$.notExpectedField", equalTo("expectedValue")));`